src/Apps/Collect2/Components/SeoProductsForCollection.tsx

```diff
@@ -0,0 +1,132 @@
+import { SeoProductsForCollection_artworks } from "__generated__/SeoProductsForCollection_artworks.graphql"
+import { Product } from "Components/v2/Seo/Product"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface SeoProductsForCollectionProps {
+  artworks: SeoProductsForCollection_artworks
+  collectionDescription: string
+  collectionURL: string
+  collectionName: string
+}
+
+// dont have anything about currency conversion in Artsy right now.
+// Just use the fixed ratio to show the approximated price.
+export function simpleConversionToUSD(price: number, currencyCode: string) {
+  if (currencyCode === "USD") {
+    return price
+  } else if (currencyCode === "GBP") {
+    return price * 1.28
+  } else if (currencyCode === "EUR") return price * 1.1
+}
+
+export const getMaxMinPrice = priceableArtworks => {
+  const priceRange = [null, null]
+  priceableArtworks.map(artwork => {
+    if (artwork.node.is_price_range && artwork.node.listPrice) {
+      const { maxPrice, minPrice } = artwork.node.listPrice
+      const convertedPrice = {
+        maxPrice: simpleConversionToUSD(maxPrice.major, maxPrice.currencyCode),
```

Comment:
I think our best bet here is to ignore currencies other than `USD` because there are tens of other currencies that we accept and could lead to very off ranges. (cc: @zephraph do you have other suggestions?)